tests/server/tso/tso_test.go

```diff
@@ -170,6 +170,46 @@ func (s *testTsoSuite) TestTsoCount0(c *C) {
 	c.Assert(err, NotNil)
 }
 
+func (s *testTsoSuite) TestRequestFollower(c *C) {
+	cluster, err := tests.NewTestCluster(s.ctx, 2)
+	c.Assert(err, IsNil)
+	defer cluster.Destroy()
+
+	err = cluster.RunInitialServers()
+	c.Assert(err, IsNil)
+	cluster.WaitLeader()
+
+	var followerServer *tests.TestServer
+	for _, s := range cluster.GetServers() {
+		if s.GetConfig().Name != cluster.GetLeader() {
+			followerServer = s
+		}
+	}
+	c.Assert(followerServer, NotNil)
+
+	grpcPDClient := testutil.MustNewGrpcClient(c, followerServer.GetAddr())
+	clusterID := followerServer.GetClusterID()
+	req := &pdpb.TsoRequest{
+		Header: testutil.NewRequestHeader(clusterID),
+		Count:  1,
+	}
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	tsoClient, err := grpcPDClient.Tso(ctx)
+	c.Assert(err, IsNil)
+	defer tsoClient.CloseSend()
+
+	start := time.Now()
+	err = tsoClient.Send(req)
+	c.Assert(err, IsNil)
+	_, err = tsoClient.Recv()
```

Comment:
I think if we could check the response to make sure the tso returned is zero, it will make this test case more rigorous.

Reply:
IMO not very useful because clients should always check `err` before using the response data. It does not really matter when err is not nil.